### ISSUE TITLE
Basic navigation example with simulated location doesn't work in 0.19.0

### ISSUE THREAD

**Issue by macdrevx** [mapbox/mapbox-navigation-ios#1590]
**Mapbox Navigation SDK version:**
0.19.0

We've noticed that simulated location isn't working when we run the basic example here: https://www.mapbox.com/ios-sdk/navigation/examples/basic/

Here's our sample project (minus access token):

[NavDemo.zip](https://github.com/mapbox/mapbox-navigation-ios/files/2271463/NavDemo.zip)

**Comment by akitchen** [comment]
I downloaded your sample project and saw what you meant. I'm not immediately sure why the example code you referenced isn't working with v0.19.x, but simulation in the Example app in the SDK codebase itself does indeed work in v0.19.2 -- if you are staying on v0.19.x for now we recommend using that for simulation/testing.

Otherwise, I updated your Demo codebase attached to use v0.22.1 of MapboxNavigation, which required a change to the Podfile as well as an update to the example view controller to match the latest example on https://www.mapbox.com/ios-sdk/navigation/examples/basic/ . I also included a script phase in the project which copies the mapbox API token in order to help avoid needing to pass it around. I've attached an archive below -- I hope it helps!

[TransLocNavDemo.zip](https://github.com/mapbox/mapbox-navigation-ios/files/2476722/TransLocNavDemo.zip)
